build/sdk/js/templates/README.mustache

```diff
@@ -118,8 +118,6 @@ The following example uses the variables `applicationClientId` and `applicationC
 ```javascript
 const sstk = require('shutterstock-api');
 
-const apiClient = sstk.ApiClient.instance;
```

Comment:
Can't remove, you still reference this a few lines later.

Reply:
Fixed, thanks for catching that.